docs/setup/appstore.md

```diff
@@ -0,0 +1,26 @@
+# App Store
+
+**Setup :material-menu-right: App Store**
+
+The **ConnexCS App Store** lets you discover, download, and install applications on your user account.
+
+The Apps created using the IDE when published and are available for use in the App Store.
+
+The Apps can be created by the other customers or developers and can be used by ConnexCS's customers.
+
+## Steps to install Apps
+
+1. Login to your account. Navigate to Setup :material-menu-right: App Store. Click on the App you wish to install.
+2. Click on **Install**.
+
+<img src= "/setup/img/appstore.png">
+
+3.After hitting the Install button. A window appears which has 2 fields, **App Name** you can either change the name of the application or keep it same. The second allows you to select the  **Version** of the application.
```

Comment:
After hitting the 'Install' button, a window appears with two fields. The first field is 'App Name', where you can either change the name of the application or keep it the same. The second field allows you to select the version of the application.

Reply:
done